src/components/NFTGem/Wrapper.vue

```diff
@@ -44,6 +44,9 @@ export default {
       }
     },
   },
+  created() {
+    this.$emit('set-gem-level', this.gemLevel);
```

Comment:
Can provide slot props for children component to access gem level

Ref:
https://github.com/likecoin/liker-land/blob/a9b5fc4ec8e9f5fcf59296027c88e9aa58f0c767/src/components/UserStats/Controller.vue#L1-L11

https://github.com/likecoin/liker-land/blob/a9b5fc4ec8e9f5fcf59296027c88e9aa58f0c767/src/components/UserStats/MyDashboard.vue#L2-L27